app/Events/Mship/Roles/RoleAssigned.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Events\Mship\Roles;
+
+use App\Events\Event;
+use App\Models\Mship\Account;
+use Illuminate\Queue\SerializesModels;
+use Spatie\Permission\Models\Role;
+
+class RoleAssigned extends Event
+{
+    use SerializesModels;
+
+    /* @var Account */
+    public $account;
```

Comment:
```suggestion
    public Account $account;
```
(Instead of the doc blocks, apply to file)

Reply:
c7744ff3